src/main/resources/digital/slovensko/autogram/server/index.html

```diff
@@ -6,13 +6,13 @@
     <link
       rel="stylesheet"
       type="text/css"
-      href="https://unpkg.com/swagger-ui-dist@4.18.1/swagger-ui.css"
+      href="assets/swagger-ui.css"
     />
   </head>
   <body>
     <div id="swagger-ui"></div>
 
-    <script src="https://unpkg.com/swagger-ui-dist@4.18.1/swagger-ui-bundle.js"></script>
+    <script src="assets/swagger-ui-bundle.js"></script>
```

Comment:
Tu by mozno stalo za to dat do nazvu aj verziu, lahsie sa to bude potom updatovat.

Reply:
Máš pravdu, ja som to len stiahol cez curl, spravím 👍😅